packages/search/src/features/registration/death/handler.test.ts

```diff
@@ -43,25 +43,6 @@ describe('Verify handlers', () => {
       mockedsearchForDuplicates.mockReturnValue([])
     })
 
-    it('should return status code 500 if invalid payload received', async () => {
```

Comment:
These were causing stderr output in test runner, so I just removed the test. There shouldn't be any expected case where the server returns a 500 status